src/sql/SqlService.ts

```diff
@@ -253,15 +251,15 @@ export class SqlServiceImpl implements SqlService {
 
     /**
      * Converts an error to HazelcastSqlException and returns it. Used by execute, close and fetch
-     * @internal
```

Comment:
Same here we had them before, these led me to add internal labels to other methods